pcdscalc/common.py

```diff
@@ -1,9 +1,35 @@
 """Module that holds common calculations."""
 import numpy as np
+from ophyd import EpicsSignal
 
 from .constants import WAVELENGTH_TO_ENERGY_LAMBDA, units
 
 
+def get_energy(energy=None):
+    """
+    Returns energy in eV.
+
+    Parameters
+    ----------
+    energy: float
+        energy in eV or keV
+    """
+    try:
+        _energy_pv = EpicsSignal('SIOC:SYS0:ML00:AO627', name='lcls_energy')
+    except:  # noqa: E722
+        _energy_pv = None
```

Comment:
The last thing I'd suggest is making this PV only once (on the first call to get_energy). Something like

```
_energy_pv = None

def get_energy(energy=None)
    global _energy_pv
    if _energy_pv is None:
        try:
```
etc.